Documentation/compilation.rst

```diff
@@ -108,9 +108,6 @@ Installing GemmForge (for GPU)
 
 .. code-block:: bash
 
-   pip install gemmforge
```

Comment:
not blue ^^. Note that there is also a comment by Carsten that you did not adress.

Reply:
Thanks, I'll fix this. 

> Note that there is also a comment by Carsten that you did not address

I've just realized that I wrote a reply but I forgot to press the button, and it turned out that I close my browser. My bad

Reply:
Hi, @Thomas-Ulrich. 

```
pip install gemmforge
```

The line is removed from the file.